images/win/scripts/Installers/Run-Antivirus.ps1

```diff
@@ -0,0 +1,33 @@
+################################################################################
+##  File:  Run-Antivirus.ps1
+##  Desc:  Run a full antivirus scan.
+##         Run right after cleanup before we sysprep
+################################################################################
+
+try{
+    Update-MpSignature
+}
+catch {
+    Write-Host "Some error was found"
+    Write-Host $_
+}
+
+if($env:run_scan_antivirus -eq $true) {
```

Comment:
please, add spaces before `(` and `{` in this file where they are missed

Reply:
Thank you, Alena. Added missed spaces